### PR TITLE
Update Zenlink UI to include amplitude and pendulum

### PR DESCRIPTION
**Pull request by ebma**
Related to [this](https://github.com/pendulum-chain/tasks/issues/3) task.

## Open Questions:
- Where to get svg icons for amplitude and pendulum? We need to have a 'circle' and a 'naked' icon for both networks each
- Which tokens should be shown in the list? ie. which tokens are available?
- Can the `asset_index` be chosen arbitrarily? The `asset_type` for assets must match what is defined [here](https://github.com/zenlinkpro/Zenlink-DEX-Module/tree/master/zenlink-protocol/src#uai-v010) but I didn't find anything about the `asset_index`

### CODE REVIEW

**Comment by ebma** [comment]
I think the icon in the list would be 'naked', and the icon at the top would be 'circle'. 
image

**Comment by RustNinja** [comment]
@ebma here is the definition of asset_index in amplitude runtime.
https://github.com/pendulum-chain/pendulum/blob/main/runtime/amplitude/src/zenlink.rs#L127-L149

in case of amplitude this is an foreign currency id 
KSM => 0
...
https://github.com/pendulum-chain/pendulum/blob/main/runtime/amplitude/src/zenlink.rs#L137

File: packages/token-lists/lists/amplitude.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "networkId": 200,
+      "address": "2124-2-516",
+      "parachainId": 2124,
+      "assetType": 2,
+      "assetIndex": 0,
+      "symbol": "KSM",
+      "decimals": 12,
+      "name": "Kusama"
+    }
```
Comment (ebma) [inline packages/token-lists/lists/amplitude.json]: So this is correct, right @RustNinja?
Do you know what the `networkId` is supposed to be?

Comment (ebma) [reply packages/token-lists/lists/amplitude.json]: (Sorry, I changed the "address" to "2124-2-0" now and force-pushed again)

Comment (RustNinja) [reply packages/token-lists/lists/amplitude.json]: @ebma 
https://github.com/paritytech/polkadot/blob/master/xcm/src/v3/junction.rs#L40-L65
or
some polkadot[.].js.org network id

Comment (RustNinja) [reply packages/token-lists/lists/amplitude.json]: @ebma seems 200 is network id for kusama ([moonriver](https://github.com/pendulum-chain/zenlink-interface/blob/e5eb0fee8800f3c68e6695f1c83ef8c3a001f490/packages/token-lists/lists/moonriver.json#L6) config and for [bifrost](https://github.com/pendulum-chain/zenlink-interface/blob/e5eb0fee8800f3c68e6695f1c83ef8c3a001f490/packages/token-lists/lists/bifrost-kusama.json#L6))

[300](https://github.com/pendulum-chain/zenlink-interface/blob/e5eb0fee8800f3c68e6695f1c83ef8c3a001f490/packages/token-lists/lists/astar.json#L6) here and here [300](https://github.com/pendulum-chain/zenlink-interface/blob/e5eb0fee8800f3c68e6695f1c83ef8c3a001f490/packages/token-lists/lists/moonbeam.json#L6) is network id for polkadot (there is value 300 for moonbeam and astar config)

Comment (ebma) [reply packages/token-lists/lists/amplitude.json]: Oh okay yeah maybe that's it

**Comment by vadaynujra** [comment]
@ebma Here are the icons:
Pendulum:
Pendulum_Token_circle
Pendulum_Token_naked

Amplitude:
Amplitude_Token_naked_aquamarine
Amplitude_Token_naked_black
Amplitude_Token_circle

**Comment by ebma** [comment]
Closing this because it was not meant to be merged.

**Pull request closed without merging**